lib/src/searchfield.dart

```diff
@@ -1,8 +1,13 @@
 import 'dart:async';
 import 'package:flutter/material.dart';
 
+enum SuggestionState {
+  /// always show the suggestions
+  enabled,
 
-enum SuggestionState {enabled,hidden}
+  /// always hide the suggestions
```

Comment:
Probably the comment should be something like 
`hide the suggestion on initial focus`